source/aws-bootstrap-kit/lib/organizational-unit.ts

```diff
@@ -44,13 +44,34 @@ export class OrganizationalUnit extends core.Construct {
                     ParentId: props.ParentId
                   }
               },
+              onUpdate: {
+                service: 'Organizations',
+                action: 'updateOrganizationalUnit',
+                physicalResourceId: cr.PhysicalResourceId.fromResponse('OrganizationalUnit.Id'),
+                region: 'us-east-1', //AWS Organizations API are only available in us-east-1 for root actions
+                parameters:
+                  {
+                    Name: props.Name,
+                    OrganizationalUnitId: new cr.PhysicalResourceIdReference()
+                  }
+              },
+              onDelete: {
+                service: 'Organizations',
+                action: 'deleteOrganizationalUnit',
+                region: 'us-east-1', //AWS Organizations API are only available in us-east-1 for root actions
+                parameters:
+                  {
+                    Name: props.Name,
+                    OrganizationalUnitId: new cr.PhysicalResourceIdReference()
```

Comment:
I'm not super familiar with custom resource usage, but I looked at the delete ou call and it looks like it needs the OU Id in order to perform the call. Is the name enough for this?